trio/_tests/test_exports.py

```diff
@@ -222,14 +235,16 @@ def no_hidden(symbols):
         from mypy.api import run
 
         # This pollutes the `empty` dir. Should this be changed?
-        run(
-            [
-                "--config-file=",
-                "--cache-dir=./.mypy_cache",
-                "-c",
-                f"import {module_name}",
-            ]
-        )
+        if not mypy_cache_updated:
```

Comment:
Throw in a `# pragma: no cover` here, as this will only be covered if `test_static_tool_sees_all_symbols` isn't run.